app.go

```diff
@@ -277,12 +277,15 @@ type PushProvider struct {
 	DisabledAt     *time.Time       `json:"disabled_at,omitempty"`
 	DisabledReason string           `json:"disabled_reason,omitempty"`
 
-	APNAuthKey string `json:"apn_auth_key,omitempty"`
-	APNKeyID   string `json:"apn_key_id,omitempty"`
-	APNTeamID  string `json:"apn_team_id,omitempty"`
-	APNTopic   string `json:"apn_topic,omitempty"`
-
-	FirebaseCredentials string `json:"firebase_credentials,omitempty"`
+	APNAuthKey              string `json:"apn_auth_key,omitempty"`
+	APNKeyID                string `json:"apn_key_id,omitempty"`
+	APNTeamID               string `json:"apn_team_id,omitempty"`
+	APNTopic                string `json:"apn_topic,omitempty"`
+	APNNotificationTemplate string `json:"apn_notification_template,omitempty"`
```

Comment:
we don't want that

Reply:
we also need to accept empty string so let's make templates pointer

Reply:
Which ones? 😄 

<img width="620" alt="image" src="https://user-images.githubusercontent.com/19969687/162259763-e4100ce1-55e0-4e6f-98f1-4d644a387802.png">

Reply:
add:
* `apn_template` to `update_app`
* `firebase_apn_template` to `push_provider`
* `firebase_notification_template` to `push_provider`